README.md

```diff
@@ -24,18 +24,18 @@
 - License
 
 ## Packages
- 
-The repository is managed as monorepo, and it contains the following packages:
+
+Dart monorepo implements different dart packages to develop fast and maintainable applications in dart and flutter.
```

Comment:
```suggestion
This dart monorepo houses the following packages to develop fast and maintainable applications for interacting with core lightning.
```
We can ignore a more general monorepo overview and only list information specific to us. Also the use of `dart` keyword is implicit and we can ignore as such. Maybe fix the message but I think we should only list important information here since its the root readme and people will notice this the first.

Reply:
Cool! doing in the next commit